src/main/java/org/resitrack/Main.java

```diff
@@ -72,4 +69,31 @@ public static void managePerson(PersonController personController, Scanner scann
 
         } while (!choice.equals(EXIT));
     }
+
+    public static void manageHouse(Scanner scanner) {
+        String choice;
+        List<House> houses = new ArrayList<>();
+        List<Town> towns = new ArrayList<>();
+
+        HouseController houseController = new HouseController();
+        TownController townController = new TownController();
```

Comment:
we created the TownController object at line 23, why we need to create it again ?

Reply:
I spent all afternoon fixing this place but it didn't work. because if I don't initialize it it won't let me access it through townController and houseController

Reply:
Do you have any ideas for this? please suggest me

Reply:
I have successfully fixed it here